From 21d535abdf138d6c55aaea5e49f7445bbd328d67 Mon Sep 17 00:00:00 2001
From: Martin Jansa <Martin.Jansa@gmail.com>
Date: Fri, 12 Apr 2013 14:18:14 +0200
Subject: [PATCH] Add config for some qemu machines

* based on om-gta02 config, most plugins are disabled (provided by
  cornucopia)
* default config prevents fsodeviced to be started by systemd, because
  both are using odeviced path on dbus bus

Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
---
 etc/qemuall/frameworkd.conf | 62 +++++++++++++++++++++++++++++++++++++++++++++
 etc/qemuarm                 |  1 +
 etc/qemux86                 |  1 +
 etc/qemux86-64              |  1 +
 4 files changed, 65 insertions(+)
 create mode 100644 etc/qemuall/frameworkd.conf
 create mode 120000 etc/qemuarm
 create mode 120000 etc/qemux86
 create mode 120000 etc/qemux86-64

diff --git a/etc/qemuall/frameworkd.conf b/etc/qemuall/frameworkd.conf
new file mode 100644
index 0000000..89f54ce
--- /dev/null
+++ b/etc/qemuall/frameworkd.conf
@@ -0,0 +1,62 @@
+# Do note that the order of plugin sections is important. Plugins that depend 
+# on others need to come later, 
+# eg. fsousage.dbus_service AFTER fsousage.lowlevel_kernel26
+
+[frameworkd]
+# indicates this configuration version, do not change
+version = 1
+# the default log_level, if not specified per subsystem or module
+# available log levels are: DEBUG, INFO, WARNING, ERROR, CRITICAL
+log_level = INFO
+# the global log_destination. Uncomment to enable
+# available destinations are: stderr, file, syslog
+log_to = file
+# if logging to a file, specify the destination
+log_destination = /var/log/frameworkd.log
+# persistence format, one of "pickle", "yaml"
+persist_format = pickle
+rootdir = ../etc/freesmartphone:/etc/freesmartphone:/usr/etc/freesmartphone
+# specify how subsystems scan for their plugins,
+# either "auto" (via filesystem scan) or "config" (via config section check)
+# the default is "auto" (slow).
+scantype = auto
+
+[odeviced]
+disable = 1
+
+[ogsmd]
+disable = 1
+
+[ogpsd]
+disable = 1
+
+[opreferencesd]
+rootdir = ../etc/freesmartphone/opreferences:/etc/freesmartphone/opreferences:/usr/etc/freesmartphone/opreferences
+
+[opreferencesd.opreferenced]
+
+[oeventsd]
+rules_file = ../etc/freesmartphone/oevents/rules.yaml:/etc/freesmartphone/oevents/rules.yaml:/usr/etc/freesmartphone/oevents/rules.yaml
+
+[oeventsd.oeventsd]
+
+[opimd]
+rootdir = ../etc/freesmartphone/opim:/etc/freesmartphone/opim:/usr/etc/freesmartphone/opim
+
+[opimd.opimd]
+
+[ousaged]
+disable = 1
+
+[ophoned]
+disable = 1
+
+[otimed]
+disable = 1
+
+[onetworkd]
+disable = 1
+
+[testing]
+disable = 1
+
diff --git a/etc/qemuarm b/etc/qemuarm
new file mode 120000
index 0000000..dd29c7a
--- /dev/null
+++ b/etc/qemuarm
@@ -0,0 +1 @@
+qemuall/
\ No newline at end of file
diff --git a/etc/qemux86 b/etc/qemux86
new file mode 120000
index 0000000..dd29c7a
--- /dev/null
+++ b/etc/qemux86
@@ -0,0 +1 @@
+qemuall/
\ No newline at end of file
diff --git a/etc/qemux86-64 b/etc/qemux86-64
new file mode 120000
index 0000000..dd29c7a
--- /dev/null
+++ b/etc/qemux86-64
@@ -0,0 +1 @@
+qemuall/
\ No newline at end of file
-- 
1.8.1.5

